plenum/common/roles.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class Roles(Enum):
```

Comment:
Please add a comment saying that these numeric constants CANNOT be changed once they have been used, because that would break backwards compatibility with the ledger.